classes/image_types_ostree.bbclass

```diff
@@ -24,10 +24,17 @@ TAR_IMAGE_ROOTFS:task-image-ostree = "${OSTREE_ROOTFS}"
 
 OSTREE_RMDIR_HELPER_MSGTYPE ?= "bbwarn"
 ostree_rmdir_helper(){
-    if ! rmdir ${1}; then
-        ${OSTREE_RMDIR_HELPER_MSGTYPE} "Data in ${1} directory is not preserved by OSTree. Consider moving it under /usr\n$(find ${1})"
-        rm -vrf ${1}
+    if [ -d ${1} ]; then
+        rmdir ${1} && return 0
+        ${OSTREE_RMDIR_HELPER_MSGTYPE} "Data in '${1}' directory is not preserved by OSTree. Consider moving it under /usr\n$(find ${1})"
+    elif [ -L ${1} ]; then
+        ${OSTREE_RMDIR_HELPER_MSGTYPE} "Data in '${1}' symbolic link is not preserved by OSTree. Consider creating it at runtime"
```

Comment:
I think the message here could be better. You're not deleting the data _in_ the symbolic link, you're just deleting the link _to_ the data.

Reply:
I will change it.

Reply:
not need anymore